tests/test_vacuum_importance_sampling.py

```diff
@@ -34,14 +34,13 @@ def test_vacuum_importance_sampling():
 
     seed = 2021
 
-    # (ytz): hacky as hell, needs to be divisible by # of hyperthreaded cores
```

Comment:
Is this comment no longer relevant? (Fixed in https://github.com/proteneer/timemachine/pull/704 , right?)

Reply:
Yeah, this has been fixed.